public/style.css

```diff
@@ -677,55 +698,95 @@ form button {
     flex: 1 100%;
   }
 
-  .closeBtn {
-    display: none;
-    padding: 10px;
-    margin: 0;
+  .item:nth-of-type(1) {
+    background-color: var(--white);
+    background-image: url(./images/imgbg.jpg);
+    background-repeat: no-repeat;
+    padding: 260px 15% 0 15%;
+    border-bottom-left-radius: 100px;
   }
 
-  .closeBtn img {
-    width: 15px;
-    float: right;
+  .achievement {
+    border: 1px solid var(--collapseborder);
+    width: 80.28%;
+    height: 500px;
+    margin: 100px auto;
+    border-radius: 25px;
+    padding: 24px;
+    background-color: var(--white);
   }
 
-  .popup {
-    width: 96%;
-    height: 96vh;
+  .achievement .fullPic {
+    width: 50%;
+    float: left;
+    background: green;
+    padding: 10px;
+    margin: 24px;
+  }
+
+  /* popup css */
+
+  /* debut section 2 */
+  .bg-popup {
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.6);
     position: fixed;
-    background-color: var(--white);
     overflow: hidden;
-    top: 2vh;
-    left: 2%;
-    z-index: 2;
-    border: 2px solid var(--collapseborder);
-    box-shadow: 0 0 10px 5px silver;
+    z-index: 1;
+    top: 0;
+  }
+
+  .popup {
+    width: 45%;
+    height: 95%;
+    background-color: white;
+    margin: 1.5% auto;
     border-radius: 8px;
-    padding: 24px;
-    margin: auto;
+    padding: 25px;
     display: flex;
+    flex-wrap: wrap;
   }
```

Comment:
- [ ] Kindly consider increasing the `width` of your `model` to match that of the Figma design template, Currently, your width is small and you have more empty space around it. It would help your design to look nicer if you increase its `width`. See below for more details:

Figma:
<img width="714" alt="Screenshot 2023-04-20 at 21 27 42" src="https://user-images.githubusercontent.com/107192076/233630723-ad052366-996c-4014-8f3a-66e6f565ea44.png">

Your design:
<img width="1197" alt="Screenshot 2023-04-21 at 11 36 02" src="https://user-images.githubusercontent.com/107192076/233630749-e7590481-71c1-49fd-90fa-49cf6f977409.png">